cinder/manifest.jsonnet

```diff
@@ -0,0 +1,193 @@
+local kpm = import "kpm.libjsonnet";
+
+function(
+  params={}
+)
+
+kpm.package({
+  package: {
+    name: "stackanetes/cinder",
+    expander: "jinja2",
+    author: "Quentin Machu",
+    version: "0.1.0",
+    description: "cinder",
+    license: "Apache 2.0",
+  },
+
+  variables: {
+    deployment: {
+      node_label: "openstack-control-plane",
```

Comment:
ditto. I'll add it to the default `parameters.yaml` so everyone can see it and edit it without searching - but even if it isn't there, any variables can be overridden by specifying a YAML file at deploy time.